Python/sysmodule.c

```diff
@@ -2603,6 +2645,9 @@ static PyMethodDef sys_methods[] = {
 #endif
     SYS__GET_CPU_COUNT_CONFIG_METHODDEF
     SYS__IS_GIL_ENABLED_METHODDEF
+#if defined(_Py_TIER2) && (_Py_TIER2 % 2 != 0)
+    SYS__JIT_ENABLED_METHODDEF
```

Comment:
AC automatically takes care of guarding the macro:

https://github.com/python/cpython/blob/4ddcd62fc47fef8a2e9773ed9ca47a30f31666fa/Python/clinic/sysmodule.c.h#L1704

So you don't need the additional `if defined(_Py_TIER2) ...` guard.

Reply:
Thanks!